src/github/geocoding.service.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpService, Injectable } from '@nestjs/common';
+import { AxiosResponse } from 'axios';
+import { GithubLocation } from './interfaces/github.interface';
+
+interface Address {
+  tourism: string;
+  house_number: string;
+  road: string;
+  quarter: string;
+  city: string;
+  state_district: string;
+  state: string;
+  postcode: string;
+  country: string;
+  country_code: string;
+}
+
+interface GeocodedResponse {
```

Comment:
These Interfaces arent quite necessary, I just add them for consistency.
They represent the Response from the Geocoding http-call.

Reply:
I prefer being strict, so these are great 👍